packages/legacy/core/App/localization/fr/index.ts

```diff
@@ -344,8 +358,13 @@ const translation = {
         "ConfirmDeclinedTitle": "Êtes-vous sûr de vouloir refuser ce justificatif?",
         "ConfirmDeclinedMessage": "Afin de recevoir à nouveau ce justificatif, vous devrez présenter une nouvelle demande auprès de l'émetteur.",
         "AbortDecline": "Non, revenir en arrière",
-        "DeclineTitle": "Refuser le justificatif",
+        "DeclineTitle": "Refuser l'offre de justificatif d'identité?",
         "Declined": "Diminué",
```

Comment:
In this context I believe "Décliné" or "Refusé" is the better term as "Diminué" means decline in the sense that it's diminishing.